src/tracepoint_format_parser.cpp

```diff
@@ -98,12 +105,26 @@ bool TracepointFormatParser::parse(ast::Program *program)
           std::ifstream format_file(format_file_path.c_str());
           if (format_file.fail())
           {
-            std::cerr << "ERROR: tracepoint not found: " << category << ":" << event_name << std::endl;
+            // Errno might get clobbered by bpftrace.error and bpftrace.warning.
+            int saved_errno = errno;
+
+            bpftrace.error(std::cerr,
+                           ap->loc,
+                           "tracepoint not found: " + category + ":" +
+                               event_name);
             // helper message:
             if (category == "syscall")
-              std::cerr << "Did you mean syscalls:" << event_name << "?" << std::endl;
+              bpftrace.warning(std::cerr,
+                               ap->loc,
+                               "Did you mean syscalls:" + event_name + "?");
+
+            errno = saved_errno;
```

Comment:
Bit strange to re-set errno. Why not just use saved_errno like `strerror(saved_errno)`?